projectsmigrator.py

```diff
@@ -509,7 +509,9 @@ def cache_after(item, after):
 def cache_after_new(item, new_status):
     # new status so append to bottom
     board, status = item['_board']
-    col = board[status]
+    col = board.get(status, None)
+    if col is None:
```

Comment:
I think if the internal representation of the target board is out of sync such that "status" is not in the board then something else has gone wrong before this point. Any idea of the scenario that might have led to this bug?

Reply:
I have a branch which I think should fix this issue another way if you want to try it out - https://github.com/pretagov/projectsmigrator/tree/fix_cached_board

Reply:
Your fix is probably more accurate, I didn't spend a lot of time digging into the details for the caching algo. I can remove my change for it and just keep the org name fix.

Reply:
There was another fix related pipeline exclusions. Can you bring that back? that was a bug unrelated to keep track of board locations

Reply:
Oops sorry I thought I'd just simplify to one fix at a time -- looks like you got it added back though, thanks!